### PR TITLE
Add support for Artifactory extra custom init containers

### PR DESCRIPTION
**Pull request by eldada**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Adding support for extra custom init containers that are useful for special dedicated bootstrapping.

**Note**: Once this PR is approved and merged, we'll add this to the other charts as well.

### CODE REVIEW

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -113,6 +113,20 @@ artifactory:
...
+  #  - name: "custom-setup"
+  #    image: "{{ .Values.initContainerImage }}"
+  #    imagePullPolicy: "{{ .Values.artifactory.image.pullPolicy }}"
+  #    command:
+  #      - 'sh'
+  #      - '-c'
+  #      - 'touch {{ .Values.artifactory.persistence.mountPath }}/example-custom-setup'
+  #    volumeMounts:
+  #      - mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+  #        name: artifactory-volume
```
Comment (jainishshah17) [inline stable/artifactory-ha/values.yaml]: @eldada I think value of this should be just `volume` https://github.com/jfrog/charts/blob/09878bed4b355e93b35433ecb0a712db2168c2ee/stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml#L214

File: stable/artifactory/values.yaml [hunk 2]
```diff
@@ -66,6 +66,19 @@ artifactory:
...
+  #  - name: "custom-setup"
+  #    image: "{{ .Values.initContainerImage }}"
+  #    imagePullPolicy: "{{ .Values.artifactory.image.pullPolicy }}"
+  #    command:
+  #      - 'sh'
+  #      - '-c'
+  #      - 'touch {{ .Values.artifactory.persistence.mountPath }}/example-custom-setup'
+  #    volumeMounts:
+  #      - mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+  #        name: artifactory-volume
```
Comment (jainishshah17) [inline stable/artifactory/values.yaml]: Same here
https://github.com/jfrog/charts/blob/09878bed4b355e93b35433ecb0a712db2168c2ee/stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml#L214

File: stable/artifactory-ha/values.yaml (hunk 1 above)
Comment (eldada) [reply stable/artifactory-ha/values.yaml]: Fixed

File: stable/artifactory/values.yaml (hunk 2 above)
Comment (eldada) [reply stable/artifactory/values.yaml]: Here it's right. No need to fix.

**Pull request merged**